examples/bri-3/components/storage/database.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
this file looks redundant to me

Reply:
Agreed, this was for the local db, before implementing Docker. This is not necessary anymore, I just forgot to remove it.